### PR TITLE
front: stdcm map adjustments

### PR DESCRIPTION
**Pull request by SarahBellaha**
Closes : [STDCM: Map adjustments](https://github.com/osrd-project/osrd-confidential/issues/673)

This PR also resolves an issue with updating the STDCM map viewport, which only refreshed once the simulation was launched.

### CODE REVIEW

File: front/src/modules/trainschedule/components/ManageTrainSchedule/ManageTrainScheduleMap/ItineraryLayer.tsx [hunk 1]
```diff
@@ -9,25 +9,29 @@ type ItineraryLayerProps = {
...
   geometry?: GeoJsonLineString;
   hideItineraryLine?: boolean;
+  showStdcmAssets: boolean;
 };
 
 export default function ItineraryLayer({
   layerOrder,
   geometry,
   hideItineraryLine = false,
+  showStdcmAssets,
```
Comment (clarani) [inline front/src/modules/trainschedule/components/ManageTrainSchedule/ManageTrainScheduleMap/ItineraryLayer.tsx:19]: You can set it by default to false ;) This way, you only need to specify it in the 2 stdcm maps
```suggestion
  showStdcmAssets = false,
```

File: front/src/applications/stdcmV2/components/StdcmConfig.tsx [hunk 2]
```diff
@@ -71,7 +69,11 @@ const StdcmConfig = ({
   const origin = useSelector(getOrigin);
   const destination = useSelector(getDestination);
 
-  const { pathfindingState } = usePathfinding(undefined, pathProperties);
+  const [pathfindingProperties, setPathfindingProperties] = useState<
+    ManageTrainSchedulePathProperties | undefined
+  >(undefined);
```
Comment (clarani) [inline front/src/applications/stdcmV2/components/StdcmConfig.tsx]: You don't need to specify that the undefined here ;) 
```suggestion
    ManageTrainSchedulePathProperties
  >();
```

**Review by clarani — COMMENTED** [review]
Almost good to me !
I think that the ui of the main map has changed on the Eex, can you check that ? 
image

I think the 1 should be in white and more centered :thinking: 
How it looks like on dev right now: 
image

**Comment by Caracol3** [comment]
Nice work  :-)
LGTM (tested)

**Review by Caracol3 — APPROVED** [review]
Nice work ! :-)
LGTM (tested)

File: front/src/modules/trainschedule/components/ManageTrainSchedule/ManageTrainScheduleMap/ItineraryLayer.tsx (hunk 1 above)
Comment (SarahBellaha) [reply front/src/modules/trainschedule/components/ManageTrainSchedule/ManageTrainScheduleMap/ItineraryLayer.tsx:19]: This value is optional in the “Map” component, before being passed to the “ItineraryLayer” component. It is already set to “false” by default and specified only for Stdcm maps :)

File: front/src/applications/stdcmV2/components/StdcmConfig.tsx [hunk 3]
```diff
@@ -71,7 +69,11 @@ const StdcmConfig = ({
   const origin = useSelector(getOrigin);
   const destination = useSelector(getDestination);
 
-  const { pathfindingState } = usePathfinding(undefined, pathProperties);
+  const [pathfindingProperties, setPathfindingProperties] = useState<
+    ManageTrainSchedulePathProperties | undefined
```
Comment (clarani) [inline front/src/applications/stdcmV2/components/StdcmConfig.tsx]: You can remove this undefined too
```suggestion
    ManageTrainSchedulePathProperties
```

**Review by clarani — APPROVED** [review]
LGTM :white_check_mark:

**Pull request merged**